lib/ar_doc_store.rb

```diff
@@ -18,6 +18,7 @@ module AttributeTypes
     autoload :EmbedsOneAttribute, "ar_doc_store/attribute_types/embeds_one_attribute"
     autoload :EmbedsManyAttribute, "ar_doc_store/attribute_types/embeds_many_attribute"
     autoload :DatetimeAttribute, "ar_doc_store/attribute_types/datetime_attribute"
+    autoload :DateAttribute, "ar_doc_store/attribute_types/date_attribute"
```

Comment:
question since I'm not totally familiar with this repo yet:  Does the date attribute need to be added to the `@mappings` in this file as well?

Reply:
Yup, good call. Must have been lost in merge. I think it might be able to infer them too though.